src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -0,0 +1,1876 @@
+//------------------------------------------------------------------------------
+/*
```

Comment:
So you didn't change any functionality when you renamed - you just pulled the class declaration into the .h file, and de-inlined all the functions. Yes?